### ISSUE TITLE
[vconsole] `"GIO_UNIMAP failed while trying to read unicode mappings: Invalid Argument`

### ISSUE THREAD

**Issue by enihcam** [systemd/systemd#28634]
### systemd version the issue has been seen with

254

### Used distribution

Archlinux

### Linux kernel version used

6.1.39-1-lts

### CPU architectures issue was seen on

x86_64

### Component

systemd, other

### Expected behaviour you didn't see

no warnings

### Unexpected behaviour you saw

warning `"GIO_UNIMAP failed while trying to read unicode mappings: Invalid Argument` from vconsole service.

### Steps to reproduce the problem

upgrade systemd to v254 and reboot

**Comment by mrc0mmand** [comment]
Can you share your `/etc/vconsole.conf`, kernel command line, and output of `localectl`?

**Comment by enihcam** [comment]
> Can you share your `/etc/vconsole.conf`, kernel command line, and output of `localectl`?

```bash
~ $ cat /etc/vconsole.conf
# This is the fallback vconsole configuration provided by systemd.

#KEYMAP=us
~ $ localectl
System Locale: LANG=en_US.UTF-8
    VC Keymap: (unset)         
   X11 Layout: (unset)         
~ $
```

**Comment by mrc0mmand** [comment]
Does this happen if you run systemd-vconsole-setup manually as well? That is, under root run:

```
# SYSTEMD_LOG_LEVEL=debug /usr/lib/systemd/systemd-vconsole-setup /dev/tty0
```

**Comment by enihcam** [comment]
and
```bash
~ $ journalctl -u systemd-vconsole-setup.service
Sep 18 22:32:18 systemd-vconsole-setup[157]: Sysfs UTF-8 flag enabled
Sep 18 22:32:18 systemd-vconsole-setup[157]: UTF-8 kbdmode enabled on /dev/tty1
Sep 18 22:32:18 systemd-vconsole-setup[157]: Executing "/usr/bin/loadkeys -q -C /dev/tty1 -u us"...
Sep 18 22:32:18 systemd-vconsole-setup[157]: Successfully forked off '(loadkeys)' as PID 161.
Sep 18 22:32:18 systemd-vconsole-setup[157]: /usr/bin/loadkeys succeeded.
Sep 18 22:32:18 systemd-vconsole-setup[157]: GIO_UNIMAP failed while trying to read unicode mappings: Invalid argument
Sep 18 22:32:18 systemd-vconsole-setup[157]: Fonts will not be copied to remaining consoles
Sep 18 22:32:19 systemd-vconsole-setup[169]: Sysfs UTF-8 flag enabled
Sep 18 22:32:19 systemd-vconsole-setup[169]: UTF-8 kbdmode enabled on /dev/tty1
Sep 18 22:32:19 systemd-vconsole-setup[169]: Executing "/usr/bin/loadkeys -q -C /dev/tty1 -u us"...
Sep 18 22:32:19 systemd-vconsole-setup[169]: Successfully forked off '(loadkeys)' as PID 172.
Sep 18 22:32:19 systemd-vconsole-setup[169]: /usr/bin/loadkeys succeeded.
Sep 18 22:32:19 systemd-vconsole-setup[169]: GIO_UNIMAP failed while trying to read unicode mappings: Invalid argument
Sep 18 22:32:19 systemd-vconsole-setup[169]: Fonts will not be copied to remaining consoles
Sep 18 22:32:19 systemd-vconsole-setup[179]: Sysfs UTF-8 flag enabled
Sep 18 22:32:19 systemd-vconsole-setup[179]: UTF-8 kbdmode enabled on /dev/tty1
Sep 18 22:32:19 systemd-vconsole-setup[179]: Executing "/usr/bin/loadkeys -q -C /dev/tty1 -u us"...
Sep 18 22:32:19 systemd-vconsole-setup[179]: Successfully forked off '(loadkeys)' as PID 181.
Sep 18 22:32:19 systemd-vconsole-setup[179]: /usr/bin/loadkeys succeeded.
Sep 18 22:32:19 systemd-vconsole-setup[179]: GIO_UNIMAP failed while trying to read unicode mappings: Invalid argument
Sep 18 22:32:19 systemd-vconsole-setup[179]: Fonts will not be copied to remaining consoles
```

**Comment by enihcam** [comment]
@keszybz Sorry for late reply. This issue still occurs.

```bash
$ systemctl status systemd-vconsole-setup
● systemd-vconsole-setup.service - Virtual Console Setup
     Loaded: loaded (/usr/lib/systemd/system/systemd-vconsole-setup.service; static)
     Active: active (exited) since Sat 2024-03-02 09:29:11 CST; 23h ago
       Docs: man:systemd-vconsole-setup.service(8)
             man:vconsole.conf(5)
    Process: 193 ExecStart=/usr/lib/systemd/systemd-vconsole-setup (code=exited, status=0/SUCCESS)
   Main PID: 193 (code=exited, status=0/SUCCESS)

Mar 02 09:29:11 systemd-vconsole-setup[193]: GIO_UNIMAP failed while trying to read unicode mappings: Invalid argument
Mar 02 09:29:11 systemd-vconsole-setup[193]: Fonts will not be copied to remaining consoles
```

systemd version info:
```bash
$ pacman -Qi systemd
Name            : systemd
Version         : 255.4-1
Description     : system and service manager
Architecture    : x86_64
URL             : https://www.github.com/systemd/systemd
Licenses        : LGPL-2.1-or-later  CC0-1.0  GPL-2.0-or-later  MIT-0
Groups          : None
Provides        : nss-myhostname  systemd-tools=255.4  udev=255.4
Depends On      : acl  libacl.so=1-64  bash  cryptsetup  libcryptsetup.so=12-64  dbus  dbus-units  kbd  kmod  libkmod.so=2-64  hwdata  libcap  libcap.so=2-64  libgcrypt  libxcrypt  libcrypt.so=2-64
                  systemd-libs  libidn2  lz4  pam  libelf  libseccomp  libseccomp.so=2-64  util-linux  libblkid.so=1-64  libmount.so=1-64  xz  pcre2  audit  libaudit.so=1-64  openssl
                  libcrypto.so=3-64  libssl.so=3-64
Optional Deps   : libmicrohttpd: systemd-journal-gatewayd and systemd-journal-remote
                  quota-tools: kernel-level quota management
                  systemd-sysvcompat: symlink package to provide sysvinit binaries
                  systemd-ukify: combine kernel and initrd into a signed Unified Kernel Image
                  polkit: allow administration as unprivileged user
                  curl: systemd-journal-upload, machinectl pull-tar and pull-raw [installed]
                  gnutls: systemd-journal-gatewayd and systemd-journal-remote [installed]
                  qrencode: show QR codes
                  iptables: firewall features [installed]
                  libbpf: support BPF programs [installed]
                  libpwquality: check password quality
                  libfido2: unlocking LUKS2 volumes with FIDO2 token
                  libp11-kit: support PKCS#11 [installed]
                  tpm2-tss: unlocking LUKS2 volumes with TPM2 [installed]
Required By     : pkgstats
Optional For    : None
Conflicts With  : nss-myhostname  systemd-tools  udev
Replaces        : nss-myhostname  systemd-tools  udev
Installed Size  : 30.77 MiB
Packager        : Christian Hesse <eworm@archlinux.org>
Build Date      : Wed 28 Feb 2024 04:38:30 AM CST
Install Date    : Wed 28 Feb 2024 10:05:35 PM CST
Install Reason  : Installed as a dependency for another package
Install Script  : Yes
Validated By    : Signature
```

**Comment by enihcam** [comment]
> We hope that this is fixed by [af189d7](https://github.com/systemd/systemd/commit/af189d7b50823876e71d1be5f5c575281ffb99c7). Please reopen if the issue still occurs with that commit (or after v255).

@keszybz can I ask for re-open? this issue still occurs. systemd version = 256.2.
